spec/unit/pretty_format_spec.rb

```diff
@@ -8,12 +8,21 @@ def method_missing(*args, &block)
     mock_action_view.send *args, &block
   end
 
-  ['hello', 23, 5.67, 10**30, :foo, Arbre::Element.new.br(:foo)].each do |obj|
+  ['hello', 23, 5.67, 10**30, :foo].each do |obj|
     it "should call `to_s` on #{obj.class}s" do
       expect(pretty_format(obj)).to eq obj.to_s
     end
   end
 
+  it "normalizes Arbre elements" do
+    expect(pretty_format(Arbre::Element.new.br)).to eq("<br>\n")
```

Comment:
This is an unexpected change in output from this patch. Before the ouput would be `<br />`, now it's `<br>`. Seems ok?

Reply:
https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br#1946446

For more information.

Reply:
Its fine. No need to be self closing in HTML5.

Reply:
Only issue would be user searching for literal `<br />` on a text string but current filter will show `<br>`... Could be confusing somehow but seems reeeeally edge...

Reply:
And an improvement anyways because previously nothing would be displayed, just some whitespace.